### DISCUSSION TITLE
Converting carbon components in @carbon/ibm-products from React to Svelte

### DISCUSSION THREAD

**Discussion by mabentley85** [carbon-design-system/carbon-components-svelte#1665] [General]
If someone were to start converting components in this repo: https://github.com/carbon-design-system/ibm-cloud-cognitive/tree/main/packages/cloud-cognitive#carbonibm-products from React to Svelte, should they be added to this repo or another repo of additional Svelte Carbon Components?

**Comment by theetrain** [comment]
Good question; since carbon-components-svelte contains 'core' components, I'm not sure how to treat IBM products or [community patterns](https://carbondesignsystem.com/community/patterns/). My instinct says they should be managed separately from here and led by an IBM team since the components you shared serve a particular product team. The maintenance effort for this project is already quite high due to the prolific number of components, so I'm not eager to add this to the scope of carbon-components-svelte at this time.

I'll need to get some direction on this. cc @metonym @tay1orjones

**Reply by metonym** [reply]
I think this is the right call; it would be better scoped as a separate project.
